### PR TITLE
#161915895 Creates a coveralls test coverage badge

### PR DESCRIPTION
**Pull request by MamboBryan**
#### What does this PR do?
This PR adds the coveralls test coverage badge 
#### Description of Task to be completed?
Read me should contain a test coverage badge
#### What are the relevant pivotal tracker stories?
#161915895

### CODE REVIEW

File: app/test/test_parcels.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    'contact': 'uncontactable',
+    'weight': '5',
+    'origin': 'Vihiga',
+    'destination': 'Nairobi',
+    'Status': 'dropped off',
+    'Price': 1250
+}
+
+
+class TestCreateApp():
```
Comment (cosmas28) [inline app/test/test_parcels.py]: document your tests and you also write the test for edge cases, i.e, when the endpoints failed to get all the parcels.
